format-parquet/src/main/java/io/cdap/plugin/format/parquet/input/ParquetInputFormatProvider.java

```diff
@@ -99,7 +99,7 @@ public Schema getDefaultSchema(FormatContext context) throws IOException {
       for (Map.Entry<String, String> entry : conf.getFileSystemProperties().entrySet()) {
         hconf.set(entry.getKey(), entry.getValue());
       }
-      final Path file = conf.getFilePathForSchemaGeneration(filePath, "parquet", hconf);
+      final Path file = conf.getFilePathForSchemaGeneration(filePath, ".+\\.parquet", hconf);
```

Comment:
why is this change needed?

Reply:
In the previous version, method `getFilePathForSchemaGeneration` could list one file from a given file system path. It provided the functionality to filter paths by extensioin (parquet and avro in that case).

In the current version, since we already use this function to list dir and pick up one file from the file system, I extended a bit the functionality to fitler files by regex rather than a pure string. Doing so, I could now pass the "file filter regex" value in this method, and get only the file that matches the regex. 

Since this method was used in the `ParquetInputFormatProvider` and `AvroInputFormatProvider` classes, I replaced the previous pure string to a regex expression.